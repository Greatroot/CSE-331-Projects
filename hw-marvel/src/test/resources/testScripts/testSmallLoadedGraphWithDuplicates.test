# Tests that a small loaded graph builds correctly with duplicate HeroModels.

LoadGraph g staffSuperheroesWithDuplicates.tsv
ListNodes g
ListChildren g Ernst-the-Bicycling-Wizard
ListChildren g Notkin-of-the-Superhuman-Beard
ListChildren g Perkins-the-Magical-Singing-Instructor
ListChildren g Grossman-the-Youngest-of-them-all
ListChildren g Ben-The-Boy