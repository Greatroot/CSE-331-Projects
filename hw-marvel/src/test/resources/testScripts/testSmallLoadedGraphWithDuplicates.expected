# Tests that a small loaded graph builds correctly with duplicate HeroModels.

loaded graph g
g contains: Ben-The-Boy Ernst-the-Bicycling-Wizard	Grossman-the-Youngest-of-them-all Notkin-of-the-Superhuman-Beard Perkins-the-Magical-Singing-Instructor
the children of Ernst-the-Bicycling-Wizard in g are: Ben-The-Boy(CSE331) Grossman-the-Youngest-of-them-all(CSE331) Notkin-of-the-Superhuman-Beard(CSE331) Notkin-of-the-Superhuman-Beard(CSE403) Perkins-the-Magical-Singing-Instructor(CSE331)
the children of Notkin-of-the-Superhuman-Beard in g are: Ben-The-Boy(CSE331) Ernst-the-Bicycling-Wizard(CSE331) Ernst-the-Bicycling-Wizard(CSE403) Grossman-the-Youngest-of-them-all(CSE331) Perkins-the-Magical-Singing-Instructor(CSE331)
the children of Perkins-the-Magical-Singing-Instructor in g are: Ben-The-Boy(CSE331) Ben-The-Boy(CSE401) Ernst-the-Bicycling-Wizard(CSE331) Grossman-the-Youngest-of-them-all(CSE331) Notkin-of-the-Superhuman-Beard(CSE331)
the children of Grossman-the-Youngest-of-them-all in g are: Ben-The-Boy(CSE331) Ernst-the-Bicycling-Wizard(CSE331) Notkin-of-the-Superhuman-Beard(CSE331) Perkins-the-Magical-Singing-Instructor(CSE331)
the children of Ben-The-Boy in g are: Ernst-the-Bicycling-Wizard(CSE331) Grossman-the-Youngest-of-them-all(CSE331) Notkin-of-the-Superhuman-Beard(CSE331) Perkins-the-Magical-Singing-Instructor(CSE331) Perkins-the-Magical-Singing-Instructor(CSE401)